mea/Cargo.toml

```diff
@@ -13,32 +13,47 @@
 # limitations under the License.
 
 [package]
+authors = ["tison <wander4096@gmail.com>"]
+categories = ["async", "synchronization", "waitgroup", "concurrency", "no-std"]
+description = """
+A runtime-agnostic library providing essential synchronization primitives for asynchronous Rust programming.
+Includes Barrier, Latch, Mutex, Semaphore, and WaitGroup implementations.
+"""
+documentation = "https://docs.rs/mea"
+keywords = ["async", "concurrency", "synchronization", "waitgroup", "mutex"]
 name = "mea"
```

Comment:
I'd prefer to list `name` and `version` at the very beginning since they are the most significant one.